src/Polly.Core.Tests/Hedging/Controller/HedgingExecutionContextTests.cs

```diff
@@ -10,7 +10,7 @@ namespace Polly.Core.Tests.Hedging.Controller;
 public class HedgingExecutionContextTests : IDisposable
 {
     private const string Handled = "Handled";
-    private static readonly TimeSpan AssertTimeout = TimeSpan.FromSeconds(5);
+    private static readonly TimeSpan _assertTimeout = TimeSpan.FromSeconds(5);
```

Comment:
Could we turn off this rule (or adjust it) to not require this for _static_ fields and for constants?